startup/preinit

```diff
@@ -50,11 +50,7 @@ for arg in $args; do
       [ -z "$ramdisk_done" ] && /opt/xensource/installer/S05ramdisk start
       ramdisk_done=1
       copy=0;;
-    mount=*)
-      [ -z "$mount_done" ] && /opt/xensource/installer/S06mount start
-      mount_done=1
-      copy=0;;
-    sshpassword=* | start=*)
+    sshpassword=* | start=* | mount=*)
```

Comment:
It would be better not to list `mount` here, then it will be copied as a command-line argument to the installer (`--mount=...`) which can be handled as part of the argument parsing code rather than frobbing the kernel command-line in readCmdlineMounts().

Reply:
It makes sense, it's tedious if you need to test using `shell` option on boot. Surely more coherent. I'll do it.